### PR TITLE
Version 3.8 prep work

### PR DESCRIPTION
**Pull request by pascalchevrel**

### CODE REVIEW

File: app/views/changelog.php [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+        <?=$relnotes('bug')?>
+        Regression: component filter is broken in string search results (flod)
+    </li>
+    <li>
+        <?=$relnotes('bug')?>
+        Regression: tabs no longer working on Health Status view (tchevalier)
+    </li>
+    <li>
+        <?=$relnotes('bug')?>
+        Regression: Fix Select button on TMX download view (tchevalier)
```
Comment (flodolo) [inline app/views/changelog.php]: "fix" (lowercase) for consistency

File: app/views/changelog.php [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+        Results are displaying HTML added by Transvision when searching for entities (tchevalier)
+    </li>
+</ul>
+
+<h3>External API changes</h3>
+<ul>
+    <li>
+        <?=$relnotes('bug')?>
+        <?=$issue(472)?>
+        Depending on the repository queried, the <var>locales</var> service returns a json feed with or without numeric keys (pascal)
```
Comment (flodolo) [inline app/views/changelog.php]: JSON

File: app/views/changelog.php [hunk 3]
```diff
@@ -1,3 +1,98 @@
...
+<ul>
+    <li>
+        <?=$relnotes('better')?>
+        <?=$issue(452)?>
+        Make Transvision code HTTPS friendly (pascal)
+    </li>
+    <li>
+        <?=$relnotes('better')?>
+        <?=$issue(455)?>
+        Add reference about apache and mod_rewrite in README file (Thegennok)
```
Comment (flodolo) [inline app/views/changelog.php]: Apache

File: app/views/changelog.php (hunk 2 above)
Comment (flodolo) [inline app/views/changelog.php]: Not sure about this one: I think you meant "returned" or "used to return", e.g. ```Depending on the repository queried, the locales service returned a JSON feed with or without numeric keys. Now returns a list of locales without keys for all repositories.```

Comment (pascalchevrel) [inline app/views/changelog.php]: yeah, I meant returned

**Pull request merged**